servicecontrol/creating-config-file.md

```diff
@@ -199,7 +208,9 @@ Valid range for this setting is from 1 hour to 200 days.
 
 #### ServiceControl/MaximumMessageThroughputPerSecond
 
-This setting was introduced in version 1.5. The setting controls the maximum throughput of messages ServiceControl will handle per second and is necessary to avoid overloading the underlying messages database. An appropriate limit ensures that the database can cope with the number of insert operations. Otherwise the query performance will drop significantly and the message expiration process will stop working under heavy insert load. Make sure to conduct thorough performance tests on the hardware before increasing this value.
+NOTE: This setting is not applicable in Version 2 and above.
```

Comment:
Is the setting ignored? Should we respect it if it is set in config?

Reply:
That setting was entirely removed so we'll ignore it from now on. The purpose of it was to ensure that the expiration policy can catch up. The tests we did with 3.5 all prove that the expiration policy can cope up with the insert load I don't think we should keep it.